ci/testnet-sanity.sh

```diff
@@ -44,23 +49,22 @@ else
   )
 fi
 
-# TODO: sanity test with validator
-# echo "--- $NET_URL: validator sanity"
-# (
-#   export USE_SNAP=1
-#   ./multinode-demo/setup.sh -t validator
-#   set -e pipefail
-#   timeout 10s ./multinode-demo/validator.sh "$NET_URL" 2>&1 | tee log
-# )
-#
-# (
-#   set +e
-#   panic=$(timeout 10s tail -f /var/snap/solana/current/validator/current | grep -C100 panic)
-#
-#   if [[ -n $panic ]]; then
-#       echo Panic observed: "$panic"
-#       exit 1
-#   fi
-# )
+echo "--- $NET_URL: validator sanity"
+if [[ -z $NO_VALIDATOR_SANITY ]]; then
```

Comment:
We can set NO_VALIDATOR_SANITY for any given testnet while not affecting the rest

Reply:
ok, but I didn't really understand why this was failing for me, or how to check whether it was working, or how to test it ;)

Reply:
we're gonna put this env var in the buildkite config?

Reply:
Yeah, it's a net-specific buildkite config variable